packages/components/date-picker/src/composables/use-basic-date-table.ts

```diff
@@ -133,9 +133,7 @@ export const useBasicDateTable = (
     const shouldIncrement = setDateText(cell, { count, rowIndex, columnIndex })
 
     const cellDate = cell.dayjs!.toDate()
-    cell.selected = _selectedDate.find(
-      (d) => d.valueOf() === cell.dayjs!.valueOf()
-    )
+    cell.selected = _selectedDate.find((d) => d.isSame(cell.dayjs, 'dates'))
```

Comment:
![image](https://github.com/element-plus/element-plus/assets/38392315/16e8c5c7-7c74-4619-a078-f8ba3a349b97)

The second param of d.isSame seems has no `dates`. Did you check it?

Reply:
The code seems to support it.

https://github.com/iamkun/dayjs/blob/2076da970047c6e0a22c8d4243a42d52833a5df2/types/index.d.ts#L26

Reply:
I found it supports plurality. What the difference between `date` and `day`?